### PR TITLE
Added benfitRating and riskRating to business struct

### PR DESCRIPTION
**Pull request by mcsearchin**
- Also updated the dataset struct docs

smartcitiesdata/smartcitiesdata/smartcitiesdata#327

co-authored-by: Jeff Grunewald <jeff@grunewalddesign.com>

### CODE REVIEW

File: lib/smart_city/dataset.ex [hunk 1]
```diff
@@ -5,8 +5,11 @@ defmodule SmartCity.Dataset do
 
   ```javascript
   const Dataset = {
-    "id": "",                  // UUID
-    "business": {              // Project Open Data Metadata Schema v1.1
+    "id": "",                // UUID
+    "business": {            // Project Open Data Metadata Schema v1.1
+      "authorEmail": "",
+      "authortName": "",
```
Comment (jmorris-pillar) [inline lib/smart_city/dataset.ex]: this is misspelled

File: lib/smart_city/dataset.ex [hunk 2]
```diff
@@ -26,17 +29,19 @@ defmodule SmartCity.Dataset do
       "publishFrequency": "",
       "referenceUrls": [""],
       "rights": "",
+      "riskRating": 0,       // value between 0.0 and 1.0
       "spatial": "",
       "temporal": ""
     },
     "technical": {
+      "allow_duplicates": true
```
Comment (NathanielScottStevens) [inline lib/smart_city/dataset.ex]: What is this field for?

Comment (mcsearchin) [reply lib/smart_city/dataset.ex]: for reaper to determine if it should de-dupe or not.

File: lib/smart_city/dataset.ex (hunk 1 above)
Comment (mcsearchin) [reply lib/smart_city/dataset.ex]: Yeah, that's like, your opinion, man.

**Pull request merged**